examples/extended/src/integration-test/groovy/spec/RegisterSpec.groovy

```diff
@@ -86,38 +93,51 @@ class RegisterSpec extends AbstractSecuritySpec {
 
 		given:
 		String un = 'test_user_abcdef' + System.currentTimeMillis()
+		when:
+		go 'register/resetPassword?t=123'
+
+		then:
+		assertHtmlContains 'Sorry, we have no record of that request, or it has expired'
 
 		when:
 		to RegisterPage
+
+        then:
```

Comment:
indentation and formatting is a bit off.  Would you be able to make it more consistent?

Reply:
Add tabs instead of spaces